plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1961,12 +1945,23 @@ elementgui.item.event_hand_tick=When item in hand tick
 elementgui.item.event_stopped_using=On player stopped using
 elementgui.item.event_entity_swings=When entity swings item
 elementgui.item.event_on_dropped=When item is dropped by player
+elementgui.item.player_useitem_finish=Player finishes using item
 elementgui.item.condition_glow=Make item glow
 elementgui.item.has_dispense_behavior=Enable custom dispense behavior
 elementgui.item.dispense_properties=Custom dispense behavior
 elementgui.item.dispense_success_condition=Dispense successfully if
 elementgui.item.dispense_result_itemstack=On dispense attempt with result
 elementgui.item.dispense_result_itemstack.default=(shrink provided itemstack if success)
+elementgui.item.food_properties=Food properties
+elementgui.item.is_food=Is a food item?
+elementgui.item.nutritional_value=Nutritional value:
+elementgui.item.saturation=Saturation:
+elementgui.item.eating_result=<html>Eating item result:<br><small>Leave empty to consume the food.
+elementgui.item.is_meat=Is this food meat?
+elementgui.item.is_edible=Is always edible?
+elementgui.item.item_animation=Item animation: 
+elementgui.item.food_3d_model=Food 3D model
```

Comment:
Where is this used, please re-check translation keys?

Reply:
Ok, but what's the problem with `trigger.player_useitem_finish` in tests?

Reply:
It seems you reference it somewhere, maybe in tooltips?

Reply:
I guess this is due to other language files... Only those files still have this key...

Reply:
Try removing the `player_useitem_finish` global trigger.

Reply:
Oh yeah, they used the same key. This is also maybe why I didn't add another key.

Reply:
I told you about it [before](https://github.com/MCreator/MCreator/pull/2002#pullrequestreview-877913417)...

Reply:
I didn't interpret the comments. Sorry 😅So what did you mean?